tools/submission/generate_final_report.py

```diff
@@ -54,7 +54,7 @@ def main():
   df['p#'] = df.apply(lambda x: int(x['host_processors_per_node']), axis=1)
 
   # details url
-  base_url = 'https://github.com/mlcommons/submissions_inference_2.1/tree/master'
+  base_url = 'https://github.com/mlcommons/submissions_inference_3.0/tree/master'
```

Comment:
Should this `base_url` point to the repo `submissions_inference_3.0` or `inference_results_v3.0`?